backend/app/controllers/charities_controller.rb

```diff
@@ -55,6 +53,10 @@ def set_charity
   end
 
   def charity_params
-    params.require(:charity).permit(:name, :description, :website_url, :logo_base64, :image_base64)
+    params.require(:charity).permit(:name, :description, :website_url)
+  end
+
+  def image_params
+    params.permit(:logo_base64, :image_base64)
   end
```

Comment:
Shouldn't this be inside :charity?

Reply:
I removed the manual `wrap_parameters` method (with the help of @ianyong LOL), so now we don't manually wrap the JSON params. This means that the wrapping is done by Rails, who will only wrap the keys found in `Model.attribute_names`. `logo_base64` and `image_base64` are not attributes, so they are not found inside the wrapped `:charity` .